plugins/tiddlywiki/tiddlyweb/syncer-actions-download-twcore.tid

```diff
@@ -0,0 +1,8 @@
+title: $:/plugins/tiddlywiki/tiddlyweb/syncer-actions/download-twcore
+tags: $:/tags/SyncerDropdown
```

Comment:
What's the purpose of presenting this menu item? Wouldn't the wiki fail to load if twcore.js was not set up correctly?

Reply:
This is optional. The purpose of the menu item "Get twcore.js" is to help users obtain a copy of twcore.js onto their local disk for their html wiki. Due to the essential boot code no longer exists as tiddlers when tiddlywiki starts up with the external-js template, the core JS needs to come from the server directly. I achieve this via an external link to `$:/core/templates/twcore.js` with the download attribute. (Is there another way?)

Ideally, I'd like to fire up the save-file dialog box directly when users click on the menu item, but the "download" behavior is inconsistent across the platforms, so I opted to direct the users to the help page instead.

Reply:
OK I think we need to handle this differently. It's too confusing for ordinary users.

Reply:
The line is only added if a node-server where an external core is active. ... IMO It's not shown for ordinary users. It's shown for users that need this option. 

There is an option "Save Snapshot for offline use" ... which only downloads the "html store", without the core ... 

That's why I did request a visible possibility to also download the core. May be "**Save Core for offline use**" would be a less confusing option text.